regression/pages/enterprise/enterprise_const.py

```diff
@@ -0,0 +1,22 @@
+"""
+URLs and constants for enterprise stuff
+"""
+
+import os
+
+
+DEFAULT_IDP_URL = u"https://idp.testshib.org/idp/Authn/UserPassword"
+
+IDP_URL = os.environ.get('IDP_URL', DEFAULT_IDP_URL)
+
+DEFAULT_IDP_NAME = "TestShib"
```

Comment:
At the moment I am using TestDemoEnterprise as default enterprise and testshib as default IDP, The values will be stored in environment variables so user can specify different values if required. In case no value is specified test will run with the default settings